apps/api/src/passport/refresh_token/src/controller.ts

```diff
@@ -70,6 +72,22 @@ export class RefreshTokenController {
     });
   }
 
+  @Put(":id")
+  @UseGuards(AuthGuard(), ActionGuard("passport:refresh-token:update"))
+  async update(@Param("id", OBJECT_ID) id: ObjectId, @Body() updates: Partial<RefreshToken>) {
```

Comment:
We should update only the disabled value if it exists; otherwise, throw an error.